BREAKING.md

```diff
@@ -12,10 +12,14 @@ There are a few steps you can take to write a good change note and avoid needing
 
 ## 0.49 Breaking changes
 - [Deprecated dirty document events and property removed from ContainerRuntime](#deprecated-dirty-document-events-and-property-removed-from-containerruntime)
+- [DeltaManager removed from fluid-framework export](#deltamanager-removed-from-fluid-framework-export)
```

Comment:
I suggest using the full package name (`@fluidframework/container-loader`) here rather than `fluid-framework`, since we publish a package with that name but it is unaffected by this change.